client/vue.config.js

```diff
@@ -1,3 +1,34 @@
 module.exports = {
   lintOnSave: false,
+  productionSourceMap: false,
```

Comment:
Please merge with the recent changes to vue.config.js

Reply:
electronBuilder plugin option added to the vue.config.js file